README.md

```diff
@@ -1,17 +1,50 @@
 # IOE Student School Allocation
 
-Public release of the code for paper 846 of AGILE2023. This is linked from the camera ready version of the paper submitted on 21 April 2023, and will be updated with additional example code, including the implimentation of the capacitated p-median location-allocation model in the spopt library in time for the AGILE conference in June 2023.
+This site demonstrates the capacitated p-median location-allocation mode developed for AGILE paper 846, Developing capacitated p-median location-allocation model in the spopt library to allow UCL student
+teacher placements using public transport.
 
-## Installation
+This repository includes the code for the "app" to create an Origin-Destination matrix from a series of school and student postcodes (app on the figure below) and an example [Jupyter Notebook](reproducible-example.ipynb) demonstrating how this is applied in practice.
+
+![](agile/mermaid-flowchart.png)
+
+This work was presented at AGILE 2023 by Nick Bearman & Levi Wolf.
+
+- [Presentation](agile/presentation.pdf)
+- [Short Paper](agile/short-paper.pdf)
+
+Currently (as at 06/06/2023) the new code developed for `spopt` is not integrated in to the main `spopt` version. There is currently a PR in progress for this at https://github.com/pysal/spopt/pull/374.
+
+## Installation of pre-processing code
+
+Install using `pip`:
+
+```python
+python -m pip install --upgrade pip
```

Comment:
You don't _really_ need to upgrade `pip`. That was a specific issue with your sysem.

Reply:
Happy for you to keep or remove as you see fit